src/assets/data/schema/schemas.json

```diff
@@ -0,0 +1,55 @@
+{
+	"schemas": [
+		{
+		"active_questionnaire_phq8_Schema": {
+			"namespace": "org.radarcns.questionnaire",
+			"type": "record",
+			"name": "Phq",
+			"doc": "Phq questionnaire",
+			"fields": [{
+					"name": "questionnaireId",
+					"type": "int",
+					"doc": "Questionnaire identifier"
+				},
+				{
+					"name": "answers",
+					"type": {
+						"type": "map",
+						"values": {
+							"name": "Answer",
+							"type": "record",
+							"doc": "Questionnaire answer",
+							"fields": [{
+									"name": "value",
+									"type": ["int", "string"],
+									"doc": "Subject answer"
+								},
+								{
+									"name": "startTime",
+									"type": "double",
+									"doc": "timestamp in UTC (s) when the question is shown"
+								},
+								{
+									"name": "endTime",
+									"type": "double",
+									"doc": "timestamp in UTC (s)  when the question is answered"
+								}
+							]
+						}
+					},
+					"doc": "Answers List"
+				},
+				{
+					"name": "startTime",
+					"type": "double",
+					"doc": "timestamp in UTC (s) when the questionnaire is submitted to the subject"
+				},
+				{
+					"name": "endTime",
+					"type": "double",
+					"doc": "timestamp in UTC (s) when the questionnaire is sent out to the platform"
+				}
+			]
+		}
+	}]
+}
```

Comment:
The `AVRO` schema should be in a `.avsc` file and stored in `RADAR-Schemas` repository. To do that on `RADAR-Schemas`, make a branch from `dev`, add your new schema under the `questionnaire` folder, make the `pull-request` to dev when you ready and ask for review.